src/main/java/de/dentrassi/rpm/builder/VerifyDetails.java

```diff
@@ -0,0 +1,159 @@
+package de.dentrassi.rpm.builder;
+
+import org.eclipse.packager.rpm.VerifyFlags;
+import org.eclipse.packager.rpm.build.FileInformation;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Each member of this class corresponds to one bit in the verify flags.
+ * See http://ftp.rpm.org/api/4.14.0/group__rpmvf.html.
+ * See https://github.com/ctron/rpm-builder/issues/41
+ */
+public final class VerifyDetails {
+   private Boolean md5;
+   private Boolean size;
+   private Boolean linkto;
+   private Boolean user;
+   private Boolean group;
+   private Boolean mtime;
+   private Boolean mode;
+   private Boolean rdev;
+   private Boolean caps;
+
+   public Boolean getMd5() {
+      return md5;
+   }
+
+   public void setMd5(Boolean md5) {
+      this.md5 = md5;
+   }
+
+   public Boolean getSize() {
+      return size;
+   }
+
+   public void setSize(Boolean size) {
+      this.size = size;
+   }
+
+   public Boolean getLinkto() {
+      return linkto;
+   }
+
+   public void setLinkto(Boolean linkto) {
+      this.linkto = linkto;
+   }
+
+   public Boolean getUser() {
+      return user;
+   }
+
+   public void setUser(Boolean user) {
+      this.user = user;
+   }
+
+   public Boolean getGroup() {
+      return group;
+   }
+
+   public void setGroup(Boolean group) {
+      this.group = group;
+   }
+
+   public Boolean getMtime() {
+      return mtime;
+   }
+
+   public void setMtime(Boolean mtime) {
+      this.mtime = mtime;
+   }
+
+   public Boolean getMode() {
+      return mode;
+   }
+
+   public void setMode(Boolean mode) {
+      this.mode = mode;
+   }
+
+   public Boolean getRdev() {
+      return rdev;
+   }
+
+   public void setRdev(Boolean rdev) {
+      this.rdev = rdev;
+   }
+
+   public Boolean getCaps() {
+      return caps;
+   }
+
+   public void setCaps(Boolean caps) {
+      this.caps = caps;
+   }
+
+   /**
+    * @see EntryDetails#apply(org.eclipse.packager.rpm.build.FileInformation)
+    * @return true if at least one flag has been set.
```

Comment:
This method doesn't seem to return anything.

Reply:
oops. Fixed.